pam/integration-tests/testdata/tapes/mandatory_password_reset.tape

```diff
@@ -0,0 +1,55 @@
+Output mandatory_password_reset.txt
+Output mandatory_password_reset.gif
```

Comment:
Sadly, yes. If we don't specify a specific output for the .gif, it generates a `out.gif` anyway. If we rely solely on it, it could create problems with the parallel testing.